content/blog/10-pearls-with-azure-functions-in-pulumi/index.md

```diff
@@ -0,0 +1,361 @@
+---
+title: "10 Pearls With Azure Functions in Pulumi"
+date: 2019-08-15
+draft: true
```

Comment:
change before publish?

Reply:
I still need a feature image... and the actual publish date

Reply:
Added a feature image